app/_partner/hp.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: partner
+lang: en
+permalink: /hp/
+
+id: hp
+name: hp
+logo:
+contact:
+
+flickr: https://www.flickr.com/
+twitter: https://twitter.com/HP
+facebook:
+hoursname:
+hourslink:
+
+
+flickr-apikey:
+flickr-setId:
+
+primary-hashtag: hp
+subhashtags: hpinspiresgiving
```

Comment:
I think this may have to be a list, even if there is just one.

Reply:
This is how it was done on the HPE site I copied & edited this from, but no clue.  Happy to take your advice!

Reply:
```
subhashtags:
  - hpinsiregiving
```

Is the correct format.